### PR TITLE
Issue/1333

### PR DESCRIPTION
**Pull request by mhaegeleMHP**

### CODE REVIEW

File: pages/collection.tsx [hunk 1]
```diff
@@ -11,28 +13,43 @@ import {
...
-import React, { useState } from 'react';
+import type { AccordionChangeEvent } from '@porsche-design-system/components-react';
+import React, { useCallback, useState } from 'react';
 import Header from '../components/header';
 import Head from 'next/head';
 
 const CollectionPage = (): JSX.Element => {
   const [submit, setSubmit] = useState(false);
   const [activePage, setActivePage] = useState(1);
+  const [isAccordion1Open, setIsAccordion1Open] = useState<boolean>(false);
```
Comment (SetupCoding) [inline pages/collection.tsx:24]: boolean generic could be removed here

File: pages/collection.tsx [hunk 2]
```diff
@@ -11,28 +13,43 @@ import {
...
   };
 
-  const handleDismiss = (e: React.MouseEvent<{}, MouseEvent>): void => {
+  const onDismiss = (e: React.MouseEvent<{}, MouseEvent>): void => {
     e.preventDefault();
     setSubmit(false);
   };
 
-  const handleActivePage = (event: CustomEvent): void => {
+  const onActivePageChange = (event: CustomEvent): void => {
```
Comment (SetupCoding) [inline pages/collection.tsx:37]: use `e` instead of `event`

File: pages/collection.tsx (hunk 1 above)
Comment (mhaegeleMHP) [reply pages/collection.tsx:24]: thats exactly the way we recommend it on our storefront

Comment (SetupCoding) [reply pages/collection.tsx:24]: There we also talked about removing it in the PR, but it was not worth the effort :D

**Pull request closed without merging**